src/plugins/vis_type_script/public/renderer/index.tsx

```diff
@@ -22,8 +23,31 @@ const getSandboxDocument = (script: string) => {
       <head>
         <meta http-equiv="content-security-policy" content="default-src none; script-src 'nonce-${nonce}' ${d3Url}">
         <script src="${d3Url}"></script>
+        <script nonce="${nonce}" type="module">
```

Comment:
I'm not sure whether we should use the same `nonce` for two inline scripts. At the same time, I can't think of a reason not to, so let's go with it for now.